.rubocop.yml

```diff
@@ -27,7 +27,7 @@ Layout/HashAlignment:
   EnforcedColonStyle: table
 
 Layout/LineLength:
-  Max: 80
+  Max: 120
```

Comment:
i  have a lot of mixed feelings on this one,  some of our namespaces are getting out of hands and make for really long lines. On the other hand, allowing longer lines would let the devs be lazy and end up being less readable code

Reply:
I would honestly say that what makes the code not readable in ruby is not longer lines, but rather block-chaining, variables named with one or two letters, double reductions, etc.
All of those things that longer lines can help reduce. And honestly 120 characters is not exactly long either.

Reply:
I agree with both sentiments. Namespacing is getting out of hand and from my POV I don't see how it could be better, as in it won't mess with readability and our brains. It annoys me to admit it but in that case, 120 seems to be the good option.
At the condition we avoid too long block chains and method length

Reply:
My concern on this is most the fact that this will end up with mixed length line in current projects. I agree that 120 could be better, but the transition will be mess. But can try.